src/controllers/recipe.controller.ts

```diff
@@ -21,7 +22,11 @@ export class RecipeController {
       };
 
       const savedRecipe = await recipeService.createRecipe(recipeData);
-      return res.status(StatusCodes.CREATED).send(savedRecipe);
+      if (savedRecipe) {
```

Comment:
Check if this if is necessary. If not then remove it (along with BAD REQUEST at the bottom).